eis/vocabs/readme.md

```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```

Comment:
Thanks for your support :). I want to publish the vocabulary but every time I try to make the pull request I get an error. I'd like to know why?